xml/admin_operating_pools.xml

```diff
@@ -64,825 +64,821 @@
   To organize data into pools, you can list, create, and remove pools. You can
   also view the usage statistics for each pool.
  </para>
- <sect1 xml:id="ceph-pools-associate">
-  <title>Associating pools with an application</title>
 
+ <sect1 xml:id="ceph-pools-operate-add-pool">
+  <title>Creating a pool</title>
   <para>
-   Before using pools, you need to associate them with an application. Pools
-   that will be used with &cephfs;, or pools that are automatically created by
-   &ogw; are automatically associated.
+   A pool can be created as either 'replicated' to recover from lost OSDs by
+   keeping multiple copies of the objects or 'erasure' to get a kind of
+   generalized RAID5/6 capability. Replicated pools require more raw storage,
```

Comment:
```
Replicated pools require more raw storage, but are less complicated and require less raw computational power. Erasure coded pools require less raw storage, but consume more memory and CPU than replicated pools because they are more complicated. [LINK_TO_HARDWARE_SECTION]
```

(Somewhere in the docs, I believe there is a discussion of how erasure coding affects memory and CPU requirements - would be nice to link to that section from here.)